cmds/exp/esxiboot/esxiboot.go

```diff
@@ -28,153 +29,57 @@
 package main
 
 import (
-	"bufio"
-	"encoding/hex"
-	"fmt"
-	"io"
+	"io/ioutil"
 	"log"
-	"os"
-	"strings"
 
 	flag "github.com/spf13/pflag"
 
 	"github.com/u-root/u-root/pkg/boot"
-	"github.com/u-root/u-root/pkg/gpt"
+	"github.com/u-root/u-root/pkg/esxi"
 )
 
-var cfg = flag.StringP("config", "c", "", "Set the ESXi config")
-var dev = flag.StringP("device", "d", "", "Set the ESXi boot device")
-
-const (
-	kernel  = "kernel"
-	args    = "kernelopt"
-	modules = "modules"
-
-	comment = '#'
-	sep     = "---"
-
-	uuidMagic = "VMWARE FAT16    "
-	uuidSize  = 32
-	partition = 5
+var (
+	cfg       = flag.StringP("config", "c", "", "ESXi config file")
+	cdrom     = flag.StringP("cdrom", "r", "", "ESXi CDROM boot device")
```

Comment:
add -r to the comments at the top of the file

Reply:
done